docs/guides/agent/agent_component_reference/begin.md

```
@@ -5,122 +5,399 @@ sidebar_custom_props: {
 
-:::tip NOTE
-The variables `MAX_CONTENT_LENGTH` in `/docker/.env` and `client_max_body_size` in `/docker/nginx/nginx.conf` set the file size limit for each upload to a dataset or RAGFlow's File system. These settings DO NOT apply in this scenario.
-:::
+#### Variable Assignor Component
+Variable Assignor writes or updates variables during workflow execution. It can save upstream results to target variables and support overwrite, clear, append and arithmetic operations for numbers, arrays and objects.
+
+Configuration:
+Add variable rules. Select target variable, choose operation type, and fill constant or reference variable as value if required. Multiple rules execute sequentially.
+
+Supported Operations:
+| Operation | Requires Value | Description |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)